python_modules/dagster/dagster_tests/daemon_tests/test_sensor_run.py

```diff
@@ -243,7 +243,7 @@ def my_pipeline_started_sensor(context):
     assert isinstance(context.instance, DagsterInstance)
 
 
-config_job = config_graph.to_job()
+config_job = config_graph.to_job(name="config_job")
```

Comment:
The necessity here was a bit surprising to me. Were we just de-duping before, and I've somehow broken that?

Reply:
Ah this is because we previously weren't catching duplicates as a result of graph coercion.

Reply:
In order to preserve existing behavior, we'll have to move job construction from sensors/schedules to the very end.